ui/js/plugins/userfas/userfas.js

```diff
@@ -28,9 +28,6 @@ define([
                 fields: [{
                     name: 'fastimezone',
                     flags: ['w_if_no_aci']
-                }, {
```

Comment:
Why did you remove the timezone?

Reply:
It was a duplicate, there's the same line right above it.

Reply:
Wouldn't it make sense to split that unrelated change out into its own commit?

Reply:
Sure.